tests/test_cli.py

```diff
@@ -101,11 +102,56 @@ def test_cli_exit_on_noinput_and_replay(mocker):
         "You can not use both replay and no_input or extra_context "
         "at the same time."
     )
+
     assert expected_error_msg in result.output
 
     mock_cookiecutter.assert_called_once_with(
         template_path,
         None,
         True,
-        replay=True
+        replay=True,
+        overwrite_if_exists=False
     )
+
+@pytest.mark.usefixtures('remove_fake_project_dir')
+def test_cli_exit_on_overwrite_if_exists_and_replay(mocker):
```

Comment:
This test is not valid anymore as we do support using both options simultaneously.